pymongoexplain/commands.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2020-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import pymongo
+from bson.son import SON
+from typing import Union
+
+Document = Union[dict, SON]
+
+class BaseCommand():
+    def __init__(self, dictionary):
+        self.command_document = self.convert_to_camelcase(dictionary)
```

Comment:
I think you should raise `NotImplementedError` here to prevent the instantiation of this class without any inheritance.